packages/editor/src/components/post-title/index.js

```diff
@@ -140,7 +140,7 @@ class PostTitle extends Component {
 								/* eslint-enable jsx-a11y/no-autofocus */
 							/>
 						</KeyboardShortcuts>
-						{ isSelected && isPostTypeViewable && <PostPermalink /> }
+						{ isSelected && isPostTypeViewable && <PostPermalink postTitle={ title } /> }
```

Comment:
Should we use `withSelect` to retrieve the title instead? We tend to avoid props forwarding when possible. For example we could imagine showing this `PostPermalink` component elsewhere where we don't have access to the title.